test_wazuh/test_fim/test_windows_registry/data/wazuh_conf_attr.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
These two configurations are pretty similar. Maybe we could use only one but well parametrized.